.github/workflows/deploy-dev.yml

```
@@ -12,17 +12,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
       - name: Read/save maven cache
-        uses: actions/cache@v2
+        uses: actions/cache@v3
         with:
           path: ~/.m2/repository
           key: ${{ runner.os }}-maven-cache-ef-personhendelse-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-maven-cache-ef-personhendelse-
             ${{ runner.os }}-
       - name: Set up JDK 17
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: '17'
```

Vi kan fjerne `actions/cache@v3` og ta i bruk cache i setup-java i stedet :) 
```suggestion
          java-version: '17'
          cache: 'maven'
```